### PR TITLE
Updates cfgov-sheer-templates to 2.1.1

### PR DESCRIPTION
**Pull request by anselmbradford**
Pulls in required markup compatibility changes in https://github.com/cfpb/cfgov-refresh/pull/2055

## Review
- @scranfill

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased][unreleased]
 ### Changed
 
+## [1.5.1] - 2016-05-23
+### Changed
+- Updated cfgov-sheer-templates dependency to get updated header JavaScript.
+
```
Comment (Scotchester) [inline CHANGELOG.md]: Please update the list at the bottom of this file, as well.

Comment (anselmbradford) [inline CHANGELOG.md]: Done

**Comment by Scotchester** [comment]
:+1:

**Comment by Scotchester** [comment]
:+1:

**Comment by ascott1** [comment]
I'm not sure if this is the culprit (or the fix), but on OaH and other sheer sites current on cf.gov the header nav appears to be missing:

screen shot 2016-05-24 at 6 54 46 am

**Comment by ascott1** [comment]
☝️ attn: @anselmbradford & @Scotchester

**Comment by Scotchester** [comment]
@ascott1 This is the fix. All fixed up and working in our dev environment. Going to production this morning!

**Comment by ascott1** [comment]
🎉 Thanks @Scotchester!

**Pull request merged**